pkg/apparmor/apparmor_linux.go

```diff
@@ -25,9 +26,34 @@ var profileDirectory = "/etc/apparmor.d"
 // IsEnabled returns true if AppArmor is enabled on the host.
 func IsEnabled() bool {
 	if unshare.IsRootless() {
+		logrus.Debug("AppAmor is not supported on rootless containers")
 		return false
 	}
-	return runcaa.IsEnabled()
+	if !runcaa.IsEnabled() {
+		logrus.Debug("AppArmor not supported by the host system")
+		return false
+	}
+
+	const (
+		binary = "apparmor_parser"
+		sbin   = "/sbin"
+	)
+
+	// `/sbin` is not always in `$PATH`, so we check it explicitly
```

Comment:
We are doing `exec.Command("apparmor_parser", "-Kr")` for loading the profiles, so we rely on $PATH only. Looking at `/sbin` can lead to false positives.

Reply:
Yep, I see. I added a new helper function `FindAppArmorParserBinary()` which can be used to obtain the parser binary path. WDYT?

Reply:
I added a new internal helper `FindAppArmorParserBinary()` to obtain the correct binary parser path, which will be passed around in the internal functions.